Automattic/woocommerce-payments
# Add e2e test for "Shopper / Subscriptions / Purchase multiple subscriptions" flow

Closes https://github.com/Automattic/woocommerce-payments/issues/2222

#### Changes proposed in this Pull Request

Title: E2E test for the "Shopper / Subscriptions / Purchase multiple subscriptions" flow.

Description: This PR adds an E2E test that covers the "Shopper / Subscriptions / Purchase multiple subscriptions" flow. It follows the steps outlined in the [testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#purchase-multiple-subscriptions).

#### Testing instructions
1. Launch the e2e environment on your local machine.
2. Run this new test: `npm run test:e2e tests/e2e/specs/shopper/shopper-subscriptions-purchase-multiple-subscriptions.spec.js`
3. The test should pass, like so:
    <img width="731" alt="Screen Shot 2021-09-05 at 9 40 52 PM" src="https://user-images.githubusercontent.com/4509348/132129210-bdd5862f-38fd-4c1d-906e-a583ed586f01.png">
